recipes/opensim/meta.yaml

```diff
@@ -0,0 +1,40 @@
+package:
+    name: opensim
+    version: "4.0.0"
+
+source:
+    git_url: https://github.com/opensim-org/opensim-core.git
+    git_rev: a59392274d4ff9ab94199485185c03c8d4abaf9d
```

Comment:
conda uses actual version numbers in it's dependency resolution, how will it work with this? In general, I think we need to have pinned version numbers that follow a pattern than can be ordered.

Reply:
Ok, I see the version number above. Shouldn't this be a git tag that points to version 4.0?

Reply:
We are preparing this conda package in advance of OpenSim's 4.0 release so that we can incorporate any necessary changes into `opensim-core` before creating the tag. We do not intend for the package to enter conda-forge until after the release.

Reply:
ok